src/verifier.ts

```diff
@@ -68,12 +70,16 @@ export class Verifier {
 
 		checkTypes.assert.nonEmptyString(options.providerBaseUrl, "Must provide the providerBaseUrl argument");
 
-		if (checkTypes.emptyArray(options.pactUrls) && !options.pactBrokerUrl) {
-			throw new Error("Must provide the pactUrls argument if no brokerUrl provided");
+		if (options.pactBrokerUrl) {
```

Comment:
I'm confused, why are we changing the name from `pactBrokerUrl` to `pactBrokerBaseUrl`?

Reply:
For consistency with all of the other languages and the CLI. Alternatively, I can just leave it as is and map to the new CLI arg?

Reply:
it was already pactBrokerUrl, leave it as is.  No need to add another.